src/library/basesqltablemodel.cpp

```diff
@@ -32,10 +34,10 @@ BaseSqlTableModel::BaseSqlTableModel(QObject* pParent,
           m_database(pTrackCollection->getDatabase()),
           m_previewDeckGroup(PlayerManager::groupForPreviewDeck(0)),
           m_iPreviewDeckTrackId(-1),
-          m_currentSearch("") {
-    m_bInitialized = false;
-    m_iSortColumn = 0;
-    m_eSortOrder = Qt::AscendingOrder;
+          m_bInitialized(false),
+          m_currentSearch(""),
+          m_trackSourceSortColumn(kIdColumn),
```

Comment:
The using code relies on this, because the first column in the track source is skipped. 
If you place a different column there the logic is messed up.